docs/src/dedicated-gen-3/security.md

```diff
@@ -1,59 +1,95 @@
 ---
 title: "Security and data privacy"
 weight: 3
-sidebarTitle: "Security and data privacy"
 description: Learn how security and data privacy are handled on {{% names/dedicated-gen-3 %}} projects.
 ---
 
+Platform.sh is committed to protecting your data and keeping your site safe, secure, and available at all times.
+All Dedicated projects are isolated and their data is fully encrypted.
+
+Should a security breach occur, Platform.sh follows a strict [security incident handling procedure](#security-incident-handling-procedure)
+to deal with the issue as promptly and efficiently as possible.
+
 ## Project isolation
 
 All Dedicated clusters are single-tenant.
-The three virtual machines are exclusively used by a single customer and each cluster is launched into its own isolated network (VPC on AWS, equivalent on other providers).
-The network is behind a firewall for incoming connections; only ports 22 (SSH), 80 (HTTP), 443 (HTTPS), 2221 (SFTP) are opened to incoming traffic.
-There are **no exceptions** for this rule, so any incoming web service requests, ETL jobs, or otherwise need to transact over one of these protocols.
+The [three virtual machines](./_index.md) are exclusively used by a single customer
+and each cluster is launched into its own isolated network (VPC on AWS, equivalent on other providers).
+
+The network is behind a firewall for incoming connections.
+Only ports 22 (SSH), 80 (HTTP), 443 (HTTPS), and 2221 (SFTP) are opened to incoming traffic.
+
+{{< note >}}
+
+There are **no exceptions** for this rule, so any incoming web service requests,
+ETL jobs, or otherwise need to transact over one of these protocols.
+
+{{</note >}}
 
 Outgoing TCP traffic isn't behind a firewall.
 Outgoing UDP traffic is disallowed.
 
-The Development Environment deploys each branch as a series of containers hosted on a shared underlying VM.
-Many customers generally share the same VM.
-All containers are allowed to connect only to other containers in their same environment, and even then only if an explicit [`relationship`](../create-apps/app-reference.md#relationships) has been defined by the user via configuration file.
+Each branch is deployed as a series of containers hosted on a shared underlying virtual machine.
+Many customers generally share the same virtual machine.
```

Comment:
I'm not sure to understand this sentence. Virtual machine are not share for more than one customer.